Content/ReleaseBriefings/release-46.tex

```diff
@@ -0,0 +1,123 @@
+\input{KokkosTutorial_PreTitle}
+\usepackage{tikz}
+\graphicspath{{4_6/figures/}}
+
+%disclaimer for Sandia. uncomment and the whole blob goes away @ b80c116300122
+\def\sandid{SANDXXXX PE}
```

Comment:
https://github.com/kokkos/Tutorials/blob/c3919ed65c2c8b34d2f8ecd977405ab9fc1e1e07/KUG-2023/beamerinnerthemekokkos.sty#L80-L88 has a similar macro for ORNL.

Reply:
done in https://github.com/kokkos/kokkos-tutorials/pull/125/commits/a486258a64b074310b8962ae83872bf83b28ec46